### PR TITLE
updating sample to the new design

### PR DESCRIPTION
**Pull request by DonMoJa**
@ldanzinger please review the updated sample when you have a moment. Thank you!

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/README.md [hunk 1]
```diff
@@ -1,19 +1,43 @@
...
-- SimpleMarkerSymbol
-- TextSymbol
-- Point
+
+1. Create a `GraphicsOverlay` for each placement mode, setting `SceneProperties::SurfacePlacement`:
+    * `Absolute`, position graphic using only its Z value.
+    * `Relative`, position graphic using its Z value plus the elevation of the surface.
+    * `DrapedBillboarded`, position graphic upright on the surface and always facing the camera, not using its z value.
+    * `DrapedFlat`, position graphic flat on the surface, not using its z value.
+2. Add graphics to the graphics overlay, `GraphicsOverlay.getGraphics.add(Graphic)`.
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/README.md]: These need to be updated to reference the Qt API patterns

File: ArcGISRuntimeSDKQt_QMLSamples/Scenes/SurfacePlacement/README.md [hunk 2]
```diff
@@ -1,19 +1,43 @@
...
-- SimpleMarkerSymbol
-- TextSymbol
-- Point
+
+1. Create a `GraphicsOverlay` for each placement mode, setting `SceneProperties.SurfacePlacement`:
+    * `Absolute`, position graphic using only its Z value.
+    * `Relative`, position graphic using its Z value plus the elevation of the surface.
+    * `DrapedBillboarded`, position graphic upright on the surface and always facing the camera, not using its z value.
+    * `DrapedFlat`, position graphic flat on the surface, not using its z value.
+2. Add graphics to the graphics overlay, `GraphicsOverlay.getGraphics.add(Graphic)`.
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_QMLSamples/Scenes/SurfacePlacement/README.md]: Same comment here as with C++ readme

File: ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/README.md (hunk 1 above)
Comment (DonMoJa) [reply ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/README.md]: Ah yes. Thank you for catching that!

**Comment by DonMoJa** [comment]
@ldanzinger both readme's have been updated.

File: ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/README.md [hunk 3]
```diff
@@ -1,19 +1,43 @@
...
-- SimpleMarkerSymbol
-- TextSymbol
-- Point
+
+1. Create a `GraphicsOverlay` for each placement mode, setting `SceneProperties::SurfacePlacement`:
+    * `Absolute`, position graphic using only its Z value.
+    * `Relative`, position graphic using its Z value plus the elevation of the surface.
+    * `DrapedBillboarded`, position graphic upright on the surface and always facing the camera, not using its z value.
+    * `DrapedFlat`, position graphic flat on the surface, not using its z value.
+2. Add graphics to the graphics overlay, `GraphicsOverlay::graphics::append(Graphic)`.
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Scenes/SurfacePlacement/README.md]: @DonMoJa 

```suggestion
2. Add graphics to the graphics overlay, `GraphicsOverlay::graphics()::append(Graphic)`.
```

**Pull request merged**